### PR TITLE
Model registration update

### PR DESCRIPTION
**Pull request by adamgilg**

### CODE REVIEW

File: model-registration.md [hunk 1]
```diff
@@ -33,50 +33,52 @@ There are two main activities associated with registering a model to Dojo:
...
-- `Model Website`: This can be a link to your model repository or another website that you may maintain that provides additional context about your model.
-- `Model Family`: Model family refers to a group of models with similar characteristics. If you have several procedures based on the same underlying model family, you can link those models or model procedures here under a common family name of your choosing, such as `LPJmL`. If your model does not have a natural grouping with other models, you can name it under an appropriate category related to your model or repeat your model name.
-- `Model Description`: Your description here is the forward-facing documentation about your model that the end-user will see. Include as much information as possible to explain what your model does and what it produces. Include any notes that may be required to explain any model idiosyncrasies. If your model takes a long time to run, you may want to include an estimated run time.
+- `Model Name`: (required) The end-user will see this name when running your model. While spaces and upper/lower case are allowed, _**do not use special characters, including parentheses**_.
```
Comment (adamgilg) [inline model-registration.md:48]: Maybe there's a better markdowney way to stylize this (required), if you think we need it at all

File: model-registration.md [hunk 2]
```diff
@@ -90,31 +92,35 @@ Steps to add a geographic coverage by building a bounding box:
...
+- Test model execution. This can be an iterative approach in that you can make several test runs to ensure that your model is running and producing the expected results.
+- Once you have a successful model run, Dojo will need to learn how you run your model, what input parameters you want to expose to the end-user, metadata about your output file(s), and the directory where your results are located.
 
 
 ### Dojo Commands:
-Take a look at the model registration commands [cheatsheet](./cheatsheet.md) for a reference for Dojo's containerization environment commands.
+Along with the Debian terminal commands you'll be able to execute in the virtual terminal as you configure your model, Dojo has a number of custom commands to help you register your model. You can type `dojo` at any time while in the terminal to bring up the helper text in the terminal, or view the [command cheatsheet](./cheatsheet.md).
```
Comment (adamgilg) [inline model-registration.md:120]: "Debian terminal commands" - bash commands? Unix commands? I'm trying to say the basic terminal commands that you can run, as we don't really mention that this is what you'll be doing anywhere else (though it is obvious, I suppose).

File: model-registration.md [hunk 3]
```diff
@@ -177,21 +183,33 @@ For a detailed description on how to do this, please go to [Data Registration](.
...
 7. Defined the location / directory of your output file(s). This is required in order to mount your model output and complete the geocoding and geotemporal transform of the results.
 
-If you have done all the above, you are ready to publish your image. Select `END SESSION`; you will be asked if you want to publish the image. Select `yes` and monitor the publication progress. When complete, you can go to https://hub.docker.com/repository/docker/jataware/dojo-publish and look under the tags section to verify that your image was pushed to DockerHub. You may need to expand the tags section.
+If you have done all the above, you are ready to review your image for publication. Select `SAVE AND CONTINUE`.
+
+
+If you want to discard the changes made to your model since launching the execution environment, you can instead select `ABANDON SESSION`. This will send you to the model's summary page. From the summary page you can relaunch the model to get back to the terminal/execution environment if you decide to make changes to the model in the future.
+
+## Summary Page & Publishing
```
Comment (adamgilg) [inline model-registration.md:203]: I'll change the weight of this

**Review by mattprintz — APPROVED** [review]
This is excellent. Thanks!

File: model-registration.md (hunk 1 above)
Comment (mattprintz) [reply model-registration.md:49]: Looks good to me.

File: model-registration.md (hunk 2 above)
Comment (mattprintz) [reply model-registration.md:121]: Maybe `Linux terminal commands` since Debian might be too restrictive.

**Pull request merged**